docs/topics/proc_add-debug-logging-to-app-vertx.adoc

```diff
@@ -51,9 +51,9 @@ The default configuration establishes a single handler on the root logger for se
 
 
 
-A JUL logging configuration file can be specified in the normal JUL way by providing a system property called: java.util.logging.config.file with the value being your configuration file.
-For more information on this and the structure of a JUL config file please consult the JUL logging documentation.
+A `java.util.logging` logging configuration file can be specified in the normal `java.util.logging` way by providing a system property called: java.util.logging.config.file with the value being your configuration file.
+For more information on this and the structure of a `java.util.logging` config file please consult the `java.util.logging` logging documentation.
```

Comment:
Maybe add a link? This one https://docs.oracle.com/javase/8/docs/technotes/guides/logging/overview.html looks reasonable

Reply:
I have removed this section but I did include the same link in the `topics/configuring-jul-logging-vertx.adoc` module